src/diffusers/models/attention_flax.py

```diff
@@ -169,6 +169,7 @@ def reshape_batch_dim_to_heads(self, tensor):
         tensor = tensor.reshape(batch_size // head_size, seq_len, dim * head_size)
         return tensor
 
+    @nn.compact
```

Comment:
There's already a `setup` method in this class, it's a bit weird (to me) that we also use `nn.compact`. Would it be feasible to add `nn.Dropout` as a module in `setup` instead?

Reply:
Yup, will add them to `setup` now.

Reply:
Addressed in 364432b5a58e988ad5a2f186c8a876148696bc62